templates/vue/src/components/TapestryNode.vue

```diff
@@ -26,12 +26,27 @@
             node.nodeType !== '' &&
             !node.hideProgress
         "
+        :x="node.coordinates.x"
+        :y="node.coordinates.y"
+        :radius="radius"
         :data-qa="`node-progress-${node.id}`"
-        :radius="node.status === 'draft' ? radius + 15 : radius"
         :progress="progress"
         :locked="!node.accessible"
-        :draft="node.status === 'draft'"
+        :status="node.status"
       ></progress-bar>
+      <dash-array
```

Comment:
Could you please name this something more meaningful like `status-bar` maybe? Also, submitted, accepted, and rejected nodes should not be dashed - they should be solid. Only simple draft nodes should have a dashed line. Please also make it one or two pixels thicker. It's a bit hard to see when the nodes are small.